src/atoms/LinkBarLinkBordered.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+import {LinkBarLink} from '..';
+
+const StyledLink = styled(LinkBarLink)`
+	border-radius: 3px;
+	border: solid 1px #9b9b9b;	
+	font-family: Cabin;
+	font-weight: 500;
+	letter-spacing: 0.4px;
+	text-align: center;
+	color: #7a7a7a;
+	margin: 0 4px;
```

Comment:
Before merging:
- Please switch from px to rem
- Please use properties from theme instead of hardcoding